nilearn/connectome/connectivity_matrices.py

```diff
@@ -284,13 +330,17 @@ class ConnectivityMeasure(BaseEstimator, TransformerMixin):
             "covariance", "precision"}, optional
         The matrix kind.
 
+    vectorize : bool, optional
```

Comment:
I'd rather have this done inside the ConnectivityMeasure object (in the transform method), rather than the function. It would minimize surprise with regards to what this function does.